DatabaseObjects/ScrumTeam.java

```diff
@@ -22,32 +23,41 @@ public ScrumTeam(ResultSet result) throws SQLException {
     @Override
     public void print() {
         System.out.println(this.scrumId + " \t" + this.projectId + "\t\"" + this.name + "\"");
+        //print the members of the team
+        for(ScrumMember temp : teamMembers) {
+            System.out.println(temp.getScrumId() + " \t" + temp.employee.getName() + "\t\"" + temp.employee.getJobTitle() + "\"");
+        }
     }
 
     @Override
     public String getTitle() {
         return this.name;
     }
 
-	public void insert(Database db) throws SQLException {
-		String query = "INSERT INTO ScrumTeam (name)"
-				+ " VALUES (?)";
-		PreparedStatement ps = db.con.prepareStatement(query, Statement.RETURN_GENERATED_KEYS);
-		ps.setString(1, this.name);
-		ps.execute();
-		ResultSet keys = ps.getGeneratedKeys();
-		if (keys.next())
-			this.scrumId = keys.getInt(1);
-		else
-			System.err.println("Did not get generated key: " + query);
-	}
+    public void insert(Database db) throws SQLException {
+        String query = "INSERT INTO ScrumTeam (name)"
+                        + " VALUES (?)";
+        PreparedStatement ps = db.con.prepareStatement(query, Statement.RETURN_GENERATED_KEYS);
+        ps.setString(1, this.name);
+        ps.execute();
+        ResultSet keys = ps.getGeneratedKeys();
+        if (keys.next())
+            this.scrumId = keys.getInt(1);
+        else
+            System.err.println("Did not get generated key: " + query);
+    }
+    
+    public void insert(ScrumMember member, int scrumEmployeeId) {
+        ScrumMember newMember = new ScrumMember(0, scrumEmployeeId);
+        teamMembers.add(newMember);
```

Comment:
so `teamMembers` is only contains members we inserted during execution?
(when we launch the program there will be a new instance of this class, so your array will be empty)